app/src/main/res/values/colors.xml

```diff
@@ -17,5 +17,5 @@
     <color name="pink">#ff4081</color>
     <color name="white">@android:color/white</color>
     <color name="reddish_pink">#ff3366</color>
-    <color name="dark">#1d1d26</color>
+    <color name="color_dark">#1d1d26</color>
```

Comment:
haha my bad, the color name should be something like `black_dark` or `blue_dark`. I don't know what color is `#1d1d26`